content/outcomes/cloud-native-development/general__dev_env_tools.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Development Environment & Tools
+weight: 20
+layout: single
+team:
+ - VMware Tanzu Labs
+---
+
+As software engineers, we will be working with our tools of the trade the whole time. Here are some of the things you will want to brush up on before we start an engagement:
+
+* **Java 8 basics** such as lambdas, the streams API, updated Date/Time libraries, default methods, and Optional.
+* Your company’s modern **build tool** of choice (Maven or Gradle). we will lean on this for both dependency management as well as building our deployable application artifacts.
+* Distributed source control with **Git**. You should be comfortable with the basic commands on the command line, including clone, fetch/merge/pull, add/rm, commit, and push.
+* Your **IDE** of choice (Eclipse with the Spring Tools Suite or JetBrains IntelliJ). While we want our application and its build process to be decoupled from your development environment, this will be the tool you use day in and day out during the engagement, so it pays to get familiar with common actions. You should be comfortable with the basics of creating and importing new projects, common code navigation, and some of the common refactoring commands. You will also want to know how to run tests within the IDE, as we will be doing that a lot.
```

Comment:
links links links links 😉

Reply:
Good idea! Most of the links are in the "homework" section and I thought it would be better not to distract while reading.

You are right, it would not harm to add more links.

c7de3a56d3b77ee9c79d345dd2c0830a0a1fd6f5